yarn.lock

```diff
@@ -23762,7 +23757,7 @@ react-docgen@^5.0.0:
     node-dir "^0.1.10"
     strip-indent "^3.0.0"
 
-"react-dom@^0.14.3 || ^15.1.0 || ^16.0.0", react-dom@^16.10.2, react-dom@^16.12.0, react-dom@^16.9.0, react-dom@^16.13.1:
+"react-dom@^0.14.3 || ^15.1.0 || ^16.0.0", react-dom@^16.10.2, react-dom@^16.12.0, react-dom@^16.13.1, react-dom@^16.9.0:
```

Comment:
Unrelated to this PR.

Looks like we had an "invalid" (working but the order was wrong) `yarn.lock`

Reply:
I think it was me who did that in #48644. I did that edit manually instead of using a tool.